# HG changeset patch
# User Laurent Gautier <lgautier@gmail.com>
# Date 1471184829 14400
# Node ID 52b73245e093d4cf32dfeb7c77e9f4da954316dc
# Parent  f272d2d75e7a9f6d5568f1471f1d7544a4d3df25
`src_dt` removed from `rpy2.robjects.lib.dplyr` (see issue #357)

diff --git a/NEWS b/NEWS
--- a/NEWS
+++ b/NEWS
@@ -8,6 +8,9 @@
   in a :class:`RuntimeError`. This is now only a warning, addressing
   issue #182.
 
+- The target version for the R package `dplyr` mapped is now 0.5.0, and
+  :func:`rpy2.robjects.lib.dplyr.src_dt` is no longer present (see issue #357)
+
 
 Release 2.8.3
 =============
diff --git a/rpy/robjects/__init__.py b/rpy/robjects/__init__.py
--- a/rpy/robjects/__init__.py
+++ b/rpy/robjects/__init__.py
@@ -236,6 +236,7 @@
 def _(obj):
     return rinterface.SexpVector([obj, ], rinterface.CPLXSXP)
 
+
 @default_converter.py2ro.register(object)
 def _(obj):
     robj = conversion.py2ri(obj)
diff --git a/rpy/robjects/lib/dplyr.py b/rpy/robjects/lib/dplyr.py
--- a/rpy/robjects/lib/dplyr.py
+++ b/rpy/robjects/lib/dplyr.py
@@ -18,7 +18,7 @@
                         version=dplyr.__version__,
                         symbol_r2python=dplyr._symbol_r2python,
                         symbol_check_after=dplyr._symbol_check_after)
-TARGET_VERSION = '0.4.3'
+TARGET_VERSION = '0.5.0'
 if dplyr.__version__ != TARGET_VERSION:
     warnings.warn('This was designed againt dplyr version %s but you have %s' % (TARGET_VERSION, dplyr.__version__))
 
@@ -190,7 +190,6 @@
 
 src_local = dplyr.src_local
 src_df = dplyr.src_df
-src_dt = dplyr.src_dt
 
 def src_sql(*args, **kwargs):
     res = dplyr.src_sql(*args, **kwargs)
